condor_watch_q.py

```diff
@@ -471,8 +475,13 @@ def table_by(clusters, attribute, abbreviate_path_components):
     rows.sort(key=lambda r: r[key])
 
     headers, rows = strip_empty_columns(rows)
+    
+    for r in rows:
```

Comment:
Make sure you edit the PR description to note that you've included this change as well.